Dayanand Viveki/HTML/ASSI1/ASSI4.html

```diff
@@ -0,0 +1,25 @@
+
+<html>
+
+<head>
+    <title>About My Self</title>
+    <style>
+        i{
+            background-color: aqua;
+            font-size: 40px;
+        }
+        b{
+            font-weight: 900;
+            font-size: 30px;
+        }
+    </style>
+</head>
+<body>
+    <div>
+        <h3>I am <u><i>Dayanand Viveki</i></u>. I belong to <b>Ahmednagar, Maharashtra</b> but recently am shifted Karve Nagar,Pune, Maharashtra.
```

Comment:
The <u> tag represents some text that is unarticulated and styled differently from normal text, such as misspelled words or proper names in Chinese text.
Instead use <ins> for inserted underlined text

Reply:
ok mdm